internal/xds/bootstrap/bootstrap.go

```diff
@@ -194,6 +211,18 @@ func (sc *ServerConfig) ServerFeatures() []string {
 	return sc.serverFeatures
 }
 
+// CallCreds returns the call credentials configuration for this server.
+func (sc *ServerConfig) CallCreds() []CallCreds {
```

Comment:
Should we call this `CallCredsConfig` and the next method `CallCreds`?

Reply:
I'm fine with removing the public methods if they are not required, which in this case seems to be the case. Less code the better :)